docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,18 +1,42 @@
 = Stackable Operator for Apache HDFS
 
-This is an operator for Kubernetes that can manage https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html[Apache HDFS] clusters.
+The Stackable Operator for https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html[Apache HDFS] is used to set up HFDS in high-availability mode. It depends on the xref:zookeeper:ROOT:index.adoc[] to operate a ZooKeeper cluster to coordinate the active/passive name node.
 
-WARNING: This operator only works with images from the https://repo.stackable.tech/#browse/browse:docker:v2%2Fstackable%2Fhadoop[Stackable] repository
+NOTE: This operator only works with images from the https://repo.stackable.tech/#browse/browse:docker:v2%2Fstackable%2Fhadoop[Stackable] repository
+
+== Roles
+
+Three xref:home:concepts:roles-and-role-groups.adoc[roles] of the HDFS cluster are implemented:
+
+* Data Node - responsible for holding the actual data. *IMPORTANT* Currently the data is kept in `hostPath` volumes of 1GB and no configuration options are exposed to the user. Each data node has its own volume.
```

Comment:
I dont understand. The user can configure (recently with https://github.com/stackabletech/hdfs-operator/pull/291 even multiple) volumes for datanodes.
```suggestion
* Data Node - responsible for storing the actual data. *IMPORTANT* Currently the data is kept in `hostPath` volumes of 1GB and no configuration options are exposed to the user. Each data node has its own volume.
```

Reply:
Maybe that is also outdated then? Do you think the comment should be removed? Maybe we can link to docs that talk about setting up the volumes?

Reply:
I have removed the statement [de6a8bd](https://github.com/stackabletech/hdfs-operator/pull/302/commits/de6a8bd9d50ff2fd3445a522faf79360f0b16409)